pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -97,39 +98,22 @@ var _ = Describe("ProjectService", func() {
 	})
 
 	Describe("enabled", func() {
-
 		When("component toggle label is set to Truthy value", func() {
```

Comment:
I think this may have to be reworded as we were previously working with string conversion.

Reply:
I guess that's still OK, as the value you can pass in the struct can be anything so good to ensure it's truthy. But since we switched from enabled to the reverse of that some text will need to change for sure.

Reply:
updated